gbfs/views.py

```diff
@@ -46,7 +48,18 @@ def gbfs(request):
 
 def gbfsSystemInformation(request):
     if request.method == "GET":
-        data = {"system_id": "TBD", "language": "TBD", "name": "TBD", "timezone": "TBD"}
+        bsp = preferences.BikeSharePreferences
+        lang = translation.get_language()
+        twoletterlangcode = lang.split("-")[0]
+        #print(dir(info.values))
+        data = {
+            "system_id": bsp.gbfs_system_id,
+            "license_url": "https://creativecommons.org/publicdomain/zero/1.0/",
+            "language": twoletterlangcode,
```

Comment:
language code in gbfs.json (line 25) has to match: "The language that will be used throughout the rest of the files. It must match the value in the system_information.json file."